src/components/EditDialogs/PurposeEditDialog/helpers.spec.js

```diff
@@ -64,13 +62,14 @@ describe('createGeojsonWithModifiedPurpose', () => {
   })
 
   it('should create the purpose in upper case', () => {
+    const timeserie = buildTimeserieWithPurpose()
     const modifiedTimeserie = createGeojsonWithModifiedPurpose({
       timeserie,
       tripId: 'tripId',
       purpose: 'shopping'
     })
 
-    expect(modifiedTimeserie.series[1].properties.manual_purpose).toBe(
+    expect(modifiedTimeserie.series[0].properties.manual_purpose).toBe(
       'SHOPPING'
     )
   })
```

Comment:
on a changé la logique des tests ici non 🤔  ? Par exemple sur tripWithNoManualPurpose. Egalement avec `to be sure other series are not impacted`

Reply:
Oui, parce que les tests portaient sur un modèle de donnée non cohérent avec ce qu'on attends :) https://github.com/cozy/coachCO2/#timeseries-models-and-nomenclature

Reply:
Il ne peut pas il y avoir d'autres series dans ce modèle de donnée, du coup pas génant de le virer :)